### ISSUE TITLE
Bot crashed after executing /status command via RPC Telegram

### ISSUE THREAD

**Issue by sshchmod777** [freqtrade/freqtrade#740]
## Step 1: Have you search for this issue before posting it?
Yes.
If you have discovered a bug in the bot, please [search our issue tracker](https://github.com/gcarq/freqtrade/issues?q=is%3Aissue). 
If it hasn't been reported, please create a new issue.

## Step 2: Describe your environment

  * Python Version: 3.6 venv
  * Branch: Develop
  * Last Commit ID:  13d6297
 
## Step 3: Describe the problem:
Bot crashes after receiving the /status or /status table command executed via telegram. It happens if there are above 2 coins currently on open status.

### Steps to reproduce:

  1. run the bot and wait for it to buy at least 2 coins.
  2. Query /status or /status table command via telegram.
  
  
### Observed Results:

1. It will respond with Stopping trader....
2. Bot will display the result but already stop.

### Relevant code exceptions or logs:

  ```
2018-05-22 21:52:22,727 - freqtrade.rpc.telegram - INFO - Executing handler: _status_table for chat_id: 500465028
2018-05-22 21:52:23,056 - freqtrade - ERROR - Fatal exception!
Traceback (most recent call last):
  File "main.py", line 45, in main
    state = freqtrade.worker(old_state=state)
  File "/home/bot/freqtrade/freqtrade/freqtradebot.py", line 120, in worker
    nb_assets=nb_assets)
  File "/home/bot/freqtrade/freqtrade/freqtradebot.py", line 132, in _throttle
    result = func(*args, **kwargs)
  File "/home/bot/freqtrade/freqtrade/freqtradebot.py", line 168, in _process
    state_changed = self.process_maybe_execute_buy()
  File "/home/bot/freqtrade/freqtrade/freqtradebot.py", line 349, in process_maybe_execute_buy
    if self.create_trade():
  File "/home/bot/freqtrade/freqtrade/freqtradebot.py", line 291, in create_trade
    (buy, sell) = self.analyze.get_signal(_pair, interval)
  File "/home/bot/freqtrade/freqtrade/analyze.py", line 119, in get_signal
    ticker_hist = get_ticker_history(pair, interval)
  File "/home/bot/freqtrade/freqtrade/exchange/__init__.py", line 35, in wrapper
    return f(*args, **kwargs)
  File "/home/bot/freqtrade/freqtrade/exchange/__init__.py", line 295, in get_ticker_history
    data_part = _API.fetch_ohlcv(pair, timeframe=tick_interval, since=since_ms)
  File "/home/bot/freqtrade/.env/lib/python3.6/site-packages/ccxt/bittrex.py", line 431, in fetch_ohlcv
    return self.parse_ohlcvs(response['result'], market, timeframe, since, limit)
  File "/home/bot/freqtrade/.env/lib/python3.6/site-packages/ccxt/base/exchange.py", line 995, in parse_ohlcvs
    ohlcv = self.parse_ohlcv(ohlcvs[i], market, timeframe, since, limit)
  File "/home/bot/freqtrade/.env/lib/python3.6/site-packages/ccxt/bittrex.py", line 411, in parse_ohlcv
    timestamp = self.parse8601(ohlcv['T'] + '+00:00')
KeyError: 'T'
2018-05-22 21:52:23,065 - freqtrade.rpc.rpc_manager - INFO - *Status:* `Stopping trader...`
2018-05-22 21:52:23,428 - freqtrade.freqtradebot - INFO - Stopping trader and cleaning up modules...
2018-05-22 21:52:23,428 - freqtrade.rpc.rpc_manager - INFO - Cleaning up rpc.telegram ...
  ```

**Comment by jblestang** [comment]
@cofibit how many pairs are you trading? Expect longer delays to process telegram commands when raising the number of pairs.

**Comment by cofibit** [comment]
@jblestang around 6-8 pairs.

**Comment by xmatthias** [comment]
i think this is fixed with #859 
does it happen with a version after this too or is the bug now solved?

**Comment by gcarq** [comment]
I think it could still happen (the chances for it being very low after #859 ), because ccxt traises a `KeyError` and not a `ccxt.ExchangeError` as we would expect, so in that case our retrier wouldn't catch this. 

That being said there isn't much more we can do other than opening an issue in ccxt.

**Comment by xmatthias** [comment]
the `KeyError` itself can still happen yes - but not when running `/status` as that result is cached (or should be, anyway)

**Comment by shusso** [comment]
feel free to re open if not fixed
